extra-resources/mods/Mythos/chinese.cardset

```diff
@@ -170,9 +155,8 @@ card('GUAN YU') {
         summon 2 of 'Terracotta Soldier' to 'you' zone 'Hand'
     }
 }
-// Info: http://en.wikipedia.org/wiki/Eight_Immortals
-// Image: http://upload.wikimedia.org/wikipedia/commons/9/9a/Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
-// License: Public Domain
+// https://en.wikipedia.org/wiki/Eight_Immortals
+// https://commons.wikimedia.org/wiki/File:Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
```

Comment:
I'm not so sure about the license of this image. The author passed away 61 years ago ([link](http://www.gutenberg.org/ebooks/15250)), which means that it's not public domain in many countries, see [List of countries' copyright lengths](https://en.wikipedia.org/wiki/List_of_countries'_copyright_lengths).

> Our ebooks may be freely used in the United States because most are not protected by U.S. copyright law, usually because their copyrights have expired. They may not be free of copyright in other countries. Readers outside of the United States must check the copyright laws of their countries before downloading or redistributing our ebooks. We also have a number of copyrighted titles, for which the copyright holder has given permission for unlimited non-commercial worldwide use.

(http://www.gutenberg.org/)